app/assets/src/components/views/compare/SamplesHeatmapView/SamplesHeatmapView.jsx

```diff
@@ -36,7 +36,11 @@ const SORT_SAMPLES_OPTIONS = [
   { text: "Cluster", value: "cluster" },
 ];
 const SORT_TAXA_OPTIONS = [
-  { text: "Genus", value: "genus" },
+  // NOTE: "genus" sort was renamed to "alphabetical" because that is currently
+  // more accurate. To sort by genus, we should sort by taxon parent ID,
+  // which is currently not always accurate, and we should show what the genus
+  // is for viruses, which is not currently shown.
+  { text: "Alphabetical", value: "genus" },
```

Comment:
Should it be alphabetical in the value too ?
Seems like it can cause confusion.